Assignment/solution_student/temp_converter/js/main.js

```diff
@@ -0,0 +1,25 @@
+var errorMessage = 'Please enter a value';
+
+$('#celsius_to_fahrenheit').on('click', function() {
+  var temp = $('#temperature').val();
+  temp = parseFloat(temp);
+  var convertedTemp = 1.8 * temp + 32;
+  console.log(convertedTemp);
+  $('#result').html("" + convertedTemp + " Fahrenheit");
+
+  if (!isNaN(temp)) {
```

Comment:
Looks like you may have left some of my code from the solution in here, but it's no big deal, the code you wrote above it is totally correct! :+1: